### ISSUE TITLE
jsonencode

### ISSUE THREAD

**Issue by MichaelBitard** [VladRassokhin/intellij-hcl#324]
### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [x] IDE version (`Help->About->Copy to Clipboard`)
    - 2020.3
 - [x] intellij-hcl plugin version (`Settings->Plugins`)
     - 0.7.10
 - [x] Terraform version (`terraform -v`)
     - v0.13.5

### Terraform Configuration Files
```hcl
resource "aws_iam_role" "lambda_role" {
  name = "any_name"

  assume_role_policy = jsonencode(
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Action": "sts:AssumeRole",
      "Principal": {
        "Service": "lambda.amazonaws.com"
      },
      "Effect": "Allow",
      "Sid": ""
    }
  ]
})
}
```

### Exception
No exception

### Expected Behavior
This jsonencode is valid and should be displayed as valid in the IDE

### Actual Behavior
See screenshot, the second level fields are displayed as red and intellij asks me to convert them to indentifier (removing the `"`).
![2021-02-08_11-43-14_grim](https://user-images.githubusercontent.com/414642/107209771-8db7a100-6a03-11eb-9edb-66cf25f5c412.png)

### Steps to Reproduce

1. Copy paste the config file anywhere in a terraform file
2. Look at all this red :)

**Comment by vmaerten** [comment]
Hello,
Any update on this issue ? :)

**Comment by capiman13** [comment]
Hello,
Any update on this issue ? :)

**Comment by VladRassokhin** [comment]
Fixed in 0.7.13
